src/vellum/workflows/nodes/displayable/bases/api_node/node.py

```diff
@@ -57,7 +57,7 @@ def _run(
             if isinstance(headers[header], VellumSecret):
                 vellum_instance = True
         if vellum_instance or bearer_token:
-            return self._vellum_execute_api(bearer_token, data, headers, method, url)
+            return self._vellum_execute_api(bearer_token, json, headers, method, url)
```

Comment:
[oos] Could we fast follow a test that would've caught this bug? We should be able to edit an existing test to verify that we are sending the correct body to Vellum